.openshift-ci/pre_tests.py

```diff
@@ -56,3 +57,15 @@ def run(self):
             check=True,
             timeout=PreSystemTests.START_PREFETCH_TIMEOUT,
         )
+
+
+class CollectionPreTest:
+    """
+    CollectionPreTest - allows finer control over collection method
+    for individual tests
```

Comment:
[nit] Probably that's effective on a job level, or can you actually switch collection back and forth per test?

```suggestion
    for individual test jobs
```